CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.4.3
```

Comment:
Rise minor or major version, since there are multiple new features added

Reply:
Bumped to 2.5.0.0